### PR TITLE
FEAT: OmnisciDB Supporting of cumulative ops through impala

### PR DESCRIPTION
**Pull request by semelianova**

### CODE REVIEW

File: ibis/impala/compiler.py [hunk 1]
```diff
@@ -170,7 +170,7 @@ def _not_null(translator, expr):
 }
 
 
-def _cumulative_to_window(translator, expr, window):
+def cumulative_to_window(translator, expr, window):
```
Comment (xmnlab) [inline ibis/impala/compiler.py]: @semelianova as this method doesn't need to be public in the API, I think you don't need to rename this function.

Comment (semelianova) [reply ibis/impala/compiler.py]: @xmnlab In this case, I get a warning that I try to access a protected member when using this method from omnisci backend (in operation.py)

Comment (xmnlab) [reply ibis/impala/compiler.py]: could you give more information about this warning pls? I tried here and I didn't see anything. is this warning raised by your IDE? as you can see there are a few more calls to `impala_compiler` functions that start with underscore inside the same module.

Comment (semelianova) [reply ibis/impala/compiler.py]: Yes, I see it from IDE

**Comment by jreback** [comment]
thanks @semelianova

File: docs/source/release.rst [hunk 2]
```diff
@@ -8,6 +8,7 @@ Release Notes
    notes for pre-1.0 versions of ibis can be found at :doc:`/release-pre-1.0`
 
 * :support:`2096` Added round() support for OmniSciDB
+* :support:`2113` Enabled cumulative ops support for OmniSciDB
```
Comment (xmnlab) [inline docs/source/release.rst]: I am wondering if it should be `feature` instead of `support` ...

**Pull request merged**